README.md

```diff
@@ -105,6 +105,31 @@ Refer to the [documentation](https://docs.microsoft.com/dotnet/api/microsoft.sql
 
 **Note:** If you are replacing an existing `.sqlproj` be sure to copy over any of these properties into the new project file.
 
+## Model compiler options
+Like `.sqlproj` projects  `MSBuild.Sdk.SqlProj` supports special `msbuild` options to control T-SQL build errors and warning.
```

Comment:
supports special `msbuild` options to control T-SQL build errors and warning. => supports controlling T-SQL build errors and warnings by using MSBuild properties.